net/utils.go

```diff
@@ -93,54 +80,3 @@ func saveKey(priv crypto.PrivKey, fp string) (err error) {
 	}
 	return f.Close()
 }
-
-func uint64ToLEB128(in uint64) []byte {
```

Comment:
why did you remove these again? they *should* save a few bytes on most messages, if that matters

Reply:
The existing solution had bugs. I thought it was better to do something simple and obvious that anyone could quickly understand. The maximum message size constant was more than 16 bits, so I went with uint32 for the size. You can think of it as having a 4-byte fixed size message header.